docs/buyers-guide.md

```diff
@@ -0,0 +1,20 @@
+# Buyer's Guide
+
+Buyers have two options:
+1. buy a complete hardware kit from the OpenPolScope project, or
+2. assemble your own kit piece by piece. 
+
+## Buy a kit from the OpenPolScope project
+ 
+- Read about the [OpenPolScope Hardware Kit](https://openpolscope.org/pages/OPS_Hardware.htm).
+- Complete the [OpenPolScope information request form](https://openpolscope.org/pages/Info_Request_Form.htm).
+
+## Buy individual components
+
```

Comment:
@talonchandler I think the 3D animation and build video will provide sufficient guidance to fellow builders. How do we share videos on GitHub?

Reply:
Sounds good. I can see the movie you uploaded to the gdrive, and I'll add it to this page.

Reply:
Uploading here. IIRC this will give me a persistent link that I can use from the buyer's guide. 

https://github.com/user-attachments/assets/a0a8bffb-bf81-4401-9ace-3b4955436b57

Reply:
thanks, @talonchandler -  do we simply upload any video to any markdown page or need to host the videos in a special way?

Reply:
As far as I can tell if you drag-and-drop upload into a comment, then it will be available to all of the markdown files via the autogenerated link. You can access the link by editing the comment.